src/lib.rs

```diff
@@ -594,7 +637,7 @@ where
 /// # Examples
 ///
 /// ```
-/// use textwrap::unfill;
+/// use textwrap::{LineEnding, unfill};
```

Comment:
I believe you wanted to add `assert_eq!(options.line_ending, LineEnding::CR)` below so that this import is used.